sdk/Lusid.Sdk.Tests/Utilities/PollyApiRetryHandlerTests.cs

```diff
@@ -315,6 +317,91 @@ public void CreateLusidFactory_WhenRetryPolicyIsAlreadyAssigned_ExistingRetryPol
 
             Assert.That(RetryConfiguration.RetryPolicy, Is.EqualTo(testPolicy));
         }
+        
+        [Test]
+        public void UsePolicyWrap_WhenCallingApiMethodHitsRateLimit_BothDefaultAndRateLimitPoliciesAreUsed()
+        {
+            const int retryAfterResponseCode = 429;
+            const int statusCodeResponseDefaultRetry = 409;
+            const int expectedNumberOfApiCalls = 6; // 3 failures for rate limit and 2 for the default one followed by success.
+            
+            // First Response
+            AddMockHttpResponseToQueue(_httpListener, statusCode: retryAfterResponseCode, responseContent: "", 
```

Comment:
it might be better to just use `HttpStatusCode.TooManyRequests` for the `statusCode`

Reply:
done, a bit of a shame that the Lusid.Sdk.csproj is still on .net standard 2 where HttpStatusCode.TooManyRequests doesn't exist. Glyn mentioned there are some reasons we can't upgrade (i don't remember what). Do we have some jira ticket to address them?